.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,16 @@
-<!-- -----------^ Click "Preview" for a functional view! -->
+## PR Checklists
+(Select the checklist that applies to your PR)
 
-## Why are you creating this Pull Request?
+Adding a Dataset Overview:
+- [ ] Dataset is available in the production VEDA STAC
+- [ ] Production STAC url is referenced in the dataset overview
+- [ ] All metadata, including those for the E&A page, is completed in the mdx file
+- [ ] Any images are compressed, ideally <500 KB
+- [ ] At least one code owner has reviewed the PR
 
-- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
-- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
-- [Other](?expand=1&template=default.md)
\ No newline at end of file
+Adding a Data Story:
+- [ ] Any data referenced in the story is already published to VEDA STAC and referenced in a separate dataset overview page (prior to merging or as part of the same PR)
+- [ ] All metadata is completed in the mdx file
+- [ ] Any images are compressed, ideally <500 KB
+- [ ] Published Date is as close to the actual release date as possible (work with veda-ui team to determine potential release date when story is close to ready)
+- [ ] At least one code owner has reviewed the PR
```

Comment:
Should we add a third one for when we only pull-in the latest `veda-ui` updates?

Reply:
@dzole0311 I like that idea! I'm not very familiar with that process though. Would you or @sandrahoang686 @hanbyul-here be able to suggest a checklist?

Reply:
oki I took a stab and pushed the change. (I also edited the doc about updating veda-ui) Let me know how it looks!